plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -1073,6 +1073,19 @@ public void testAlterTableWithUnsupportedProperties()
         assertUpdate("DROP TABLE " + tableName);
     }
```

Comment:
Can we have tests with a combination of `change_data_feed_enabled`, `ALTER_TABLE`, and `writer_version`?

Reply:
Something like `testCreateTableWithCdfEnabledAndUnsupportedWriterVersionFails`, `testCreateTableAsWithCdfPropertyWriterVersionNotUpgraded`

Reply:
I modified the test I added and renamed it to `testSettingChangeDataGeedEnabledProperty`